### PR TITLE
Fix issue in which --kind appends to the default choices value

### PR DESCRIPTION
**Pull request by AndresPineros**
Fixes https://github.com/hjacobs/kube-downscaler/issues/55

### CODE REVIEW

**Comment by hjacobs** [comment]
Thanks! Can you change the option to `--include-resources` like I have in `kube-janitor`?

**Comment by AndresPineros** [comment]
Sure, I'll fix the linting too

File: README.rst [hunk 1]
```diff
@@ -123,7 +123,7 @@ Available command line options:
 ``--namespace``
     Namespace (default: all namespaces)
 ``--kind``
```
Comment (hjacobs) [inline README.rst]: needs to be updated, too

File: kube_downscaler/cmd.py [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 import argparse
 
+VALID_RESOURCES = ["deployments", "statefulsets", "stacks"]
```
Comment (hjacobs) [inline kube_downscaler/cmd.py]: you can use frozenset here already

File: kube_downscaler/cmd.py [hunk 3]
```diff
@@ -2,6 +2,14 @@
...
+
+
+def check_include_resources(value):
+    resources = frozenset(value.split(','))
+    assert resources.issubset(frozenset(VALID_RESOURCES)), "--include-resources argument should contain a subset of {}".format(VALID_RESOURCES)
```
Comment (hjacobs) [inline kube_downscaler/cmd.py]: you can use `<=` instead of "issubset"

**Pull request merged**